data/mods/Aftershock/items/ammo/25mm.json

```diff
@@ -0,0 +1,24 @@
+[
+  {
+    "id": "afs_25mm_shot",
+    "type": "AMMO",
+    "name": { "str_sp": "25mm canister shot" },
+    "description": "25mm caseless rounds.  The most common type of ammo used for light grenade launchers.",
+    "weight": "130 g",
+    "volume": "230 ml",
+    "price": 225,
+    "price_postapoc": 8000,
```

Comment:
Of note I just went through all items in this PR, and the alien battery, but it might be worth while to go looking over items in aftershock and due an audit in the future to remove the post_apoc prices.